mypy.ini

```diff
@@ -0,0 +1,63 @@
+[mypy]
```

Comment:
what do you think about using setup.cfg, to not introduce a new config file?

Reply:
That works for me :ok_hand:

Reply:
Updated in 9eaaa1ea1dc2481577dca7dce15da139c79d0699